hr_skill/__manifest__.py

```diff
@@ -20,7 +20,7 @@
 
```

Comment:
Hi @feketemihai,
What do you mean by "update with small headers"? I don't see any python filename with big headers.
Thanks